tests/server/test_resource_list.py

```diff
@@ -248,19 +252,23 @@ async def test_filter_resources(server, client, env_with_resources):
     assert result.code == 200
     assert len(result.result["data"]) == 1
 
+    result = await client.resource_list(env.id, filter={"resource_id_value": ["file"]})
+    assert result.code == 200
+    assert len(result.result["data"]) == 4
+
     result = await client.resource_list(env.id, filter={"resource_id_value": ["/etc/file", "/tmp/file"]})
     assert result.code == 200
     assert len(result.result["data"]) == 4
 
-    result = await client.resource_list(env.id, filter={"resource_type": ["Directory"]})
+    result = await client.resource_list(env.id, filter={"resource_type": ["NullResource"]})
     assert result.code == 200
-    assert len(result.result["data"]) == 2
+    assert len(result.result["data"]) == 6
```

Comment:
Hmm now that all resources in the `env_with_resources` fixture are of the same type we no longer test the filtering by resource_type and I think we should